src/metatrain/utils/data/readers/ase.py

```diff
@@ -4,6 +4,21 @@
 import ase.io
 import torch
 from metatensor.torch import Labels, TensorBlock
+from metatensor.torch.atomistic import System, systems_to_torch
+
+
+def read_systems_ase(filename: str, dtype: torch.dtype = torch.float32) -> List[System]:
```

Comment:
I would change the default here to float64, or have no default at all

Reply:
EDIT: is this for the next PR?

Reply:
Yes I do it after this PR! But it is almost top on my list.